examples/regression/1-quickstart/plot_compare_conformity_scores.py

```diff
@@ -0,0 +1,197 @@
+:class:`mapie.conformity_scores.AbsoluteConformityScore` which
+considers the absolute residuals as the conformity scores and
+:class:`mapie.conformity_scores.GammaConformityScore` which
+considers the residuals divided by the predicted means as conformity scores.
+We consider the standard CV+ resampling method.
+
+We would like to emphasize one main limitation with this example.
+With the default conformity score, the confidence intervals
+are approximately equal over the range of house prices which may
+be inapporpriate when the price range is wide. The Gamma conformity score
+overcomes this issue by considering confidence intervals with width
+proportional to the predicted mean. For low prices, the Gamma confidence
+intervals are narrower than the default ones, conversely to high prices
+for which the conficence intervals are higher but visually more relevant.
+The empirical coverage is similar between the two conformity scores.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sklearn.datasets import fetch_openml
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.model_selection import train_test_split
+
+from mapie.conformity_scores import GammaConformityScore
+from mapie.metrics import regression_coverage_score
+from mapie.regression import MapieRegressor
+
+np.random.seed(0)
+
+# Parameters
+features = [
+    "MSSubClass",
+    "LotArea",
+    "OverallQual",
+    "OverallCond",
+    "GarageArea",
+]
+alpha = 0.05
+rf_kwargs = {"n_estimators": 10, "random_state": 0}
+
+##############################################################################
+# 1. Load dataset with a target following approximativeley a Gamma distribution
+# --------------------------------------------------------------------------
+#
+# We start by loading a dataset with a target following approximately
+# a Gamma distribution. The GammaConformityScore is relevant in such cases.
+# Two sub dataset are extracted: the training and test ones.
+
+X, y = fetch_openml(name="house_prices", return_X_y=True)
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X[features], y, test_size=0.2
+)
+
+##############################################################################
+# 2. Train model with two conformity scores
+# --------------------------------------------------------------------------
+#
+# Two models are trained with two different conformity score:
+# - AbsoluteConformityScore (default conformity score) relevant for target
+#   positive as well as negative. The confidence interval widths are in this
+#   case approximately the same over the range of prediction.
+# - GammaConformityScore relevant for target following roughly a Gamma
+#   distribution. The confidence interval widths scale with the predicted
+#   value.
+
+# Train model with AbsoluteConformityScore
+mapie = MapieRegressor(RandomForestRegressor(**rf_kwargs))
+mapie.fit(X_train, y_train)
+y_pred_absconfscore, y_pis_absconfscore = mapie.predict(X_test, alpha=[alpha])
```

Comment:
A float value for `alpha` is accepted by MAPIE :-)

Reply:
Done